### PR TITLE
Update gnn example

### PR DESCRIPTION
**Pull request by ZiyueXu77**
Fixes # .

### Description

For env preparation steps, torch geometric needs special treatment

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/advanced/gnn/README.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ Install additional requirements:
 ```
 python3 -m pip install -r requirements.txt
 ```
+To further support PyTorch Geometric, we need to install in the following manner, please check the torch version and perform the installation accordingly:
+```
+pip install torch-scatter torch-sparse torch-cluster torch-spline-conv torch-geometric -f https://data.pyg.org/whl/torch-2.1.0+cpu.html
```
Comment (YuanTingHsieh) [inline examples/advanced/gnn/README.md:34]: suggest change this to the official documentation about installation:
https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html

File: examples/advanced/gnn/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
+nvflare
+torch
+tensorboard
 scikit_learn
```
Comment (IsaacYangSLA) [inline examples/advanced/gnn/requirements.txt:4]: Thanks.  I agree that we might need to quote the steps in the official installation document if it's available.

File: examples/advanced/gnn/README.md (hunk 1 above)
Comment (ZiyueXu77) [reply examples/advanced/gnn/README.md]: good point, thanks! updated

File: examples/advanced/gnn/requirements.txt (hunk 2 above)
Comment (ZiyueXu77) [reply examples/advanced/gnn/requirements.txt:5]: sounds good! moved main torch_geometric here

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM

**Pull request merged**